apis/components/v1alpha1/datasciencepipelines/types.go

```diff
@@ -0,0 +1,23 @@
+// +kubebuilder:object:generate=true
+
+// Package datasciencepipelines provides a set of types used for DataSciencePipelines component
+package datasciencepipelines
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+type PreloadedPipelinesSpec struct {
+	// Configures whether to automatically import the InstructLab pipeline.
+	// You must enable the trainingoperator component to run the InstructLab pipeline.
+	InstructLab PreloadedPipelineOptions `json:"instructLab,omitempty"`
```

Comment:
@HumairAK @etirelli should we add some logic in the operator to handle this requirement ? i.e. reporting the component as _misconfigured_ ?

Reply:
no,  any validation/requirement checking will happen as an initial step within the pipeline itself